packages/server/src/v1/pulse/application/services/activities.service.ts

```diff
@@ -138,7 +137,7 @@ export class ActivitiesService {
       operatingSystem: dto.operating_system || '',
       machine: dto.machine || '',
       userAgent: dto.user_agent || '',
-      time: DateTime.fromMillis(dto.time as number).toISO(),
+      time: DateTime.fromMillis((dto.time as number) * 1000).toISO(),
```

Comment:
Can we make it so time comes in as an ISO date since that is what we want to save it as? I ran into this bug with the browser extension and changed it to send a milli second time stamp, when the original code was sending the UNIX timestamp. 

If we want to do UNIX timestamps in, that is fine. I'll do a quick fix for the browser.

Reply:
We should keep with what wakatime is using as a starting point so we can have compatibility with other wakatime extensions that we aren't forking at this time.

Reply:
Okay, I will put up a PR in the browser then too

Reply:
https://github.com/CodeClimbersIO/browser-extension/pull/18